boot/bootdata/livecd.inf

```diff
@@ -41,6 +41,16 @@ HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32",,0x00000012
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\LastVisitedMRU",,0x00000012
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\OpenSaveMRU",,0x00000012
 
+; Policies overrides
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies",,0x00000012
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer",,0x00000012
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer","NoDisconnect",0x00010003,1
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer","NoLogoff",0x00010003,1
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer","StartMenuLogoff",0x00010003,1
+HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","DisableLockWorkstation",0x00010003,1
+HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System","DisableLockWorkstation",0x00010003,1
+HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System","DisableChangePassword",0x00010003,1
```

Comment:
Since these policies are UI only, I wonder if the system would (and supposed to) block such attempts in minwin mode?

Reply:
Maybe there are more modules that check also for these values, but I haven't investigated. For now I have added the handling (see #8372) only in msgina.